src/lib/dbManager.js

```diff
@@ -37,6 +37,7 @@ function DBManager(options){
 
 	this.localDbName = (typeof options === 'string') ? options : options.localDbName;
 	this.remoteDbName = options.remoteDbName ? options.remoteDbName : this.localDbName;
+	this.localDbPath = options.localDbPath ? options.localDbPath : this.localDbPath;
```

Comment:
I think this line should be
`this.localDbPath = options.localDbPath ? options.localDbPath : env.dbPath;`

`this.localDbPath` should always be undefined at this point, so we should grab it's default value from env.